geograypher/cameras/cameras.py

```diff
@@ -690,46 +705,69 @@ def get_lon_lat_coords(self):
         """Returns a list of GPS coords for each camera"""
         return [x.get_lon_lat() for x in self.cameras]
 
+    def get_camera_locations(self):
```

Comment:
This method should probably allow the user to choose between having the z component or not. If you don't want to keep track of the args for the `Camera` method, you could have use `**kwargs` so the user can pass keyword arguments that will be passed to the `Camera.get_camera_location(**kwargs)` method. This function should also have a docstring, but you can refer to the method it calls rather than being explicit if needed.